ldap/3.2/modules/ROOT/pages/ldap-connector-reference.adoc

```diff
@@ -709,23 +709,18 @@ and define a single element list with the value.
 
 Performs an LDAP search and streams the result to the rest of the flow.
 
-This means that instead of returning a list with all results, this operation
-partitions the LDAP search result into pages or an individual entry if
-resultPageSize is 1, or lists of size resultPageSize.
+This means that instead of returning a list with all results, if the LDAP server supports paging, this operation partitions the LDAP search result into pages or lists, the size of which are specified by the Page Size parameter. If the Page Size value is set to 1, results are returned as an individual entry.
 
-This is an intercepting operation, which means that for each
-result, or an individual entry if resultPageSize is 1 or List of
-resultPageSize size, the rest of the flow executes. Each
-execution returns a result that aggregates
-into a List of results.
+This is an intercepting operation, which means that for each search
+result, the rest of the flow executes. Each execution returns a result that aggregates into a List of results.
```

Comment:
Not for now, but in the future, I would try to clarify what "the rest of the flow" executes. The rest of the flow from where?